client/src/components/SeasonCollectionImageCardComponent/seasoncollectionimg.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './seasoncollectionimg.css'
```

Comment:
Can you be consistent with casing? I.e. the names of your files should be camel case, like `SeasonCollectionImage`. It's also important to not abbreviate words